File: katas/7-kyu/balanced-number-special-numbers-series-number-1/solution.js
function balancedNum(number)
{
let intArr = Array.from(String(number),num=>+num)
return intArr.slice(0, Math.ceil(intArr.length/2)-1).reduce((acc,x)=>acc+x,0) === intArr.slice(Math.floor(intArr.length/2 +1)).reduce((acc,x)=>acc+x,0) ? "Balanced" : "Not Balanced" 
}

File: katas/7-kyu/balanced-number-special-numbers-series-number-1/solution.py
def balanced_num(number):
    int_arr = list(map(int, str(number)))  
    length = len(int_arr)
    
    left_sum = sum(int_arr[:(length // 2) - (1 if length % 2 == 0 else 0)])
    right_sum = sum(int_arr[(length // 2) + 1:])
    
    return "Balanced" if left_sum == right_sum else "Not Balanced"
